src/interfaces/task.ts

```diff
@@ -0,0 +1,24 @@
+import { MultiPolygon, Polygon } from '@turf/turf';
```

Comment:
Probably should be one of our's as well

Reply:
i don't think it should be ours, turf is peer dependency and structure isn't likely to be changed